internal/features/features.go

```diff
@@ -40,4 +40,9 @@ const (
 	// details.
 	// https://github.com/crossplane/crossplane/blob/f32496bed53a393c8239376fd8266ddf2ef84d61/design/design-doc-composition-validating-webhook.md
 	EnableAlphaCompositionWebhookSchemaValidation feature.Flag = "EnableAlphaCompositionWebhookSchemaValidation"
+
+	// EnableAlphaUsages enables alpha support for deletion ordering and
```

Comment:
Good point. However, I prefer to keep it as is because:
- It is only in the codebase, not in our API, e.g. the flag that users need to pass, which is `--enable-usages`.
- We have [two different](https://github.com/crossplane/docs/pull/532/files#diff-2afc226346e34d233dbbcfe60db218b152021a6af20f9b6119ac376656a725daR13-R14)use cases for `Usage` API and deletion ordering is one of them.